nupic/utils.py

```diff
@@ -83,3 +83,12 @@ def next(self, newValue):
 
   def getSlidingWindow(self):
     return self.slidingWindow
+
+
+  def __setstate__(self, state):
+    self.__dict__.update(state)
+
+    if not hasattr(self, "total"):
+      self.total = 0
+      self.slidingWindow = []
```

Comment:
If the serialized version had a sliding window do you really want to reset it? How about replace these lines with:

```
self.total = sum(self.slidingWindow)
```